scrapers.js

```diff
@@ -1548,13 +1589,24 @@ const scrapers = [
       // Resource contains multiple updates shown chronologically however it is unclear now that
       // they will follow any reliable pattern. This captures the first one as the latest
 
+      if (datetime.scrapeDateIsBefore('2020-3-17')) {
+        const cases = parse.number(
+          $('font:contains("Glenn County COVID-19 Cases")')
+            .first()
+            .text()
+            .match(/Cases: (\d+)/)[1]
+        );
+        return {
+          cases
+        };
+      }
+
       const cases = parse.number(
-        $('font:contains("Glenn County COVID-19 Cases")')
+        $('span:contains("Glenn County COVID-19 Cases")')
```

Comment:
You could put a conditional in the selector if you wanted:

```suggestion
        let tagName = datetime.scrapeDateIsBefore('2020-3-17') ? 'font' : 'span';
        $(tagName + ':contains("Glenn County COVID-19 Cases")')
```

The other options is to just do  `$(':contains("Glenn County COVID-19 Cases")').last()` -- but it's probably slower... Thoughts?